docs/src/projects/delete-project.md

```diff
@@ -36,4 +36,11 @@ title=Using the CLI
 {{< /codetabs >}}
 
 You are billed only for the portion of a month when the project was active.
-A user account with no projects has no charges.
+
+When you remove all your projects from your user account,
+your subscription is canceled and you're not charged.
```

Comment:
:lady_beetle: This entirely precise. A subscription is the old term for a plan, which corresponds to one project. I wouldn't keep the term subscription now.